apps/crn-frontend/src/network/working-groups/__tests__/api.test.ts

```diff
@@ -78,7 +78,7 @@ describe('getWorkingGroups', () => {
     );
   });
 });
-describe('teamResearchOutput', () => {
```

Comment:
Do we use this API file since the hooks are common?

Reply:
Not all of them are common, we do have get working group by id and get team by id.